cal/cal/static/js/app.js

```diff
@@ -111,29 +111,41 @@ analyticsApp.component('tags', {
 function TagsDetailCtrl($scope, $http) {
   var tagUrl = '/v1/tags/' + this.tagId + '/events';
   var eventweek = '/v1/tags/' + this.tagId + '/eventWeek';
+  var query_timezone = moment.tz.guess();
   $scope.tagDetails = [];
   $scope.tagEvents = [];
+  $scope.tagHours = this.tagHours;
```

Comment:
Changing it to `this.tagDetails = []` and `this.tagDetails.push` gives an angular error saying Cannot read property 'push' of undefined... am I doing it wrong, do I have to do more?